src/data/pagesdata/global/invoice.json

```
@@ -0,0 +1,235 @@
+        },
+        {
+          "value": "85%",
+          "label": "Improvement in Financial Visibility"
+        },
+        {
+          "value": "100%",
+          "label": "Data Security & Uptime Guarantee"
+        }
+      ]
+    },
+    "cta": {
```

Biome (2.1.2)

[error] The key cta was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)